active-object/README.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: pattern
+title: Active Object
+folder: active-object
+permalink: /patterns/active-object/
+categories: Concurrency
+tags:
+ - Gang of Four
```

Comment:
Active Object is not part of GoF patterns. I would suggest `Performance` tag
https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags

Reply:
The tag hasn't been updated yet